release/cloudbuild_withsecure_signature.yaml

```diff
@@ -0,0 +1,41 @@
+# This Cloud Build trigger copies the WithSecure signature for a certain
+# Trusted OS release version to the bucket (and "subdir") that contains the
+# Trusted OS as built by transparency.dev and the detached signature as signed
+# by transparency.dev.
+#
+# This is the second Cloud Build trigger for a given release. The first should
+# have already created the Trusted OS elf file and the transparency.dev
+# detached signature.
+#
+# The Trusted OS elf should only be used if both signatures are verified
+# successfully.
+#
+#### WithSecure Expectations ####
+#
+# WithSecure is expected to overwrite the _WITHSECURE_SIG_FILE and
```

Comment:
Did you consider having a directory of all the previous signatures and have the release version be part of the file name (e.g. `/release/withsecure/trstued_os_v1.5.3.sig`)?

I'm thinking that it might reduce complexity a touch without the `release_version.txt` indirection, and would help avoid any raciness between processes at GCB and WS ends?

Reply:
It did cross my mind, but the thing that's a bit unfortunate is that Cloud Build isn't smart enough to know which new signature has been added, so we'll have to do more logic in the Cloud Build config. Maybe we could re-write every version's signature every time but 1. the bash logic seems to already be on the edge of what Cloud Build is intended for in terms of complexity and 2. we may not want to rewrite signatures, because that seems not WAI? TL;DR It's possible to do that with more complexity (perhaps another GCF function or another binary like `json_constructor` in the applet?) but I'm not sure if it's worth it (yet). WDYT?

Reply:
Does GCB know which version tag triggered it - if so perhaps we could pass that in as a var so it can build the expected filename?

Reply:
Spoke offline, I have updated the PR to actually have versioned WithSecure signatures.